ipcore/dataplane_config.py

```diff
@@ -21,10 +22,24 @@ def __init__(self):
         self.sink_max_channel = 0
         self.source_max_channel = 0
 
-    def populate_additional_filesets(self, additionalFilesetAbsDir):
-        for filename in os.listdir(additionalFilesetAbsDir):
-            if filename.endswith(".vhd") and '_avalon' not in filename and (self.name in filename or "pkg" in filename):
-                self.additional_filesets.append(filename)
+    def populate_additional_filesets(self, additionalFilesetAbsDir, sourceFilePatterns):
+        for pattern in sourceFilePatterns:
+            
+            subdirectories = pattern.split('\\\\')[0:-1]
```

Comment:
Does using `\` make this Windows-specific? If so, obviously not a big deal right now as nobody uses Linux except me.

Reply:
Its been a hot minute since I actually wrote this but I believe I went with '\' instead of '/' because '/' was not playing well with the regex stuff. It definitely does make it Windows-specific for now.